builder/builder.go

```diff
@@ -1151,6 +1151,46 @@ func (b *Builder) EditBundles(bundles []string, suppressEditor bool, add bool, g
 	return nil
 }
 
+// ValidateBundles runs bundle parsing validation on a list of local bundles, or
+// all local bundles if allLocal is true. In addition to parsing errors, errors
+// are generated if the bundle is not found in local-bundles.
+func (b *Builder) ValidateBundles(bundles []string, allLocal bool) error {
```

Comment:
Optional: I've found the allLocal parameter here kind of odd. I'd try either one of those: (a) a separate function take lists the local bundles (and we pass that)  (b) a separate ValidateLocalBundles() that takes no arguments  (c) read the list of bundles in the call site and pass that.

Reply:
Let me do a second pass at this logic to see if I can refactor in a way that feels better.